styles.css

```diff
@@ -29,25 +29,49 @@ body {
 
 .logo__circle_color_dark-blue { background: #0056ec; }
 
-.main-content__title {
-  font-family: 'pt_serif', serif;
+.main {
+  padding: 0 56% 100px 0;
+  background: top right no-repeat url(img/img.svg);
 }
 
+.main__title {
+  margin: 0 0 40px 0;
+  font: bold 50px/66px 'pt_serif', arial, helvetica, sans-serif;
+}
+
+.main__title-text_highlight {
+  background: #ecf3ff;
+}
+
+.main__description {
+  margin: 0 0 41px 0;
+  font: 18px/21px Roboto, arial, helvetica, sans-serif;
+}
+
+.main__button {
+  display: inline-block;
+  font: 14px 16px Roboto, arial, helvetica, sans-serif;
+  color: #fff;
+  background: #0056ec;
+  padding: 14px 16px;
+  text-decoration: none;
+  border-radius: 4px;
+  transition: opacity .5s;
+}
+
+ .main__button:hover {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6255586)